### PR TITLE
Fixed problem of suggestion box 

### PR DESCRIPTION
**Pull request by saurabh28march**
Also  added  extrenal css and js file

### CODE REVIEW

**Comment by saurabh28march** [comment]
Related to issue #8 
Please review @maze-runnar

File: templates/index.html [hunk 1]
```diff
@@ -4,162 +4,30 @@
...
-	</style>
+       
 	   
+<script>
+  function myFunction() {
+    var x = document.getElementById("snackbar");
+    x.className = "show";
+    setTimeout(function(){ x.className = x.className.replace("show", ""); }, 20000);
+  }
+</script>
```
Comment (maze-runnar) [inline templates/index.html:26]: if possible add this also in external js file

Comment (saurabh28march) [reply templates/index.html:26]: ok

**Comment by maze-runnar** [comment]
@SARTHAK1SINGH please merge!

**Pull request merged**